R/class-workbook-wrappers.R

```diff
@@ -61,9 +65,14 @@ wb_workbook <- function(
 #' }
 wb_save <- function(wb, path = NULL, overwrite = TRUE) {
   assert_workbook(wb)
-  wb$clone()$save(path = path, overwrite = overwrite)$path
+  wb$clone()$save(path = path, overwrite = overwrite)
 }
 
+#' @rdname wb_save
+#' @export
```

Comment:
`wb_write()` is a bit cluttering.

Can remove and suggest: `wb_save(wb)$path` or include `wb_get_path()` (needs `wbWorkbook$get_path()`.